internal/services/iothub/iothub_fallback_route_resource.go

```diff
@@ -44,6 +45,19 @@ func resourceIotHubFallbackRoute() *pluginsdk.Resource {
 				ValidateFunc: validate.IoTHubName,
 			},
 
+			"source": {
+				Type:     pluginsdk.TypeString,
+				Required: true,
```

Comment:
we can't just add this as required as it is a breaking change, we should make it optional with the default value used below: `devices.RoutingSourceDeviceMessages`